exporter/signalfxexporter/translation/converter.go

```diff
@@ -102,121 +102,123 @@ func (c *MetricsConverter) MetricDataToSignalFxV2(md consumerdata.MetricsData) (
 	return
 }
 
-func (c *MetricsConverter) metricDataToSfxDataPoints(md consumerdata.MetricsData) (
+func (c *MetricsConverter) metricDataToSfxDataPoints(mds []consumerdata.MetricsData) (
 	sfxDataPoints []*sfxpb.DataPoint,
 	numDroppedTimeSeries int,
 ) {
 	var err error
 
-	// Labels from Node and Resource.
-	// TODO: Options to add lib, service name, etc as dimensions?
-	//  Q.: what about resource type?
-	nodeAttribs := md.Node.GetAttributes()
-	resourceAttribs := md.Resource.GetLabels()
-	numExtraDimensions := len(nodeAttribs) + len(resourceAttribs)
-	var extraDimensions []*sfxpb.Dimension
-	if numExtraDimensions > 0 {
-		extraDimensions = make([]*sfxpb.Dimension, 0, numExtraDimensions)
-		extraDimensions = appendAttributesToDimensions(extraDimensions, nodeAttribs)
-		extraDimensions = appendAttributesToDimensions(extraDimensions, resourceAttribs)
-	}
-
-	for _, metric := range md.Metrics {
-		if metric == nil || metric.MetricDescriptor == nil {
-			c.logger.Warn("Received nil metrics data or nil descriptor for metrics")
-			numDroppedTimeSeries += len(metric.GetTimeseries())
-			continue
+	for _, md := range mds {
```

Comment:
Can we put this loop outside of `metricDataToSfxDataPoints`, in `MetricDataToSignalFxV2` for example. it'll make the diff cleaner at least

Reply:
Updated.